images/twitter.html

```diff
@@ -0,0 +1,124 @@
+                    alt="faceprofile">
+            </div>
```

Comment:
not sure about this part, but you could turn this section into a footer  to maintain that space created by the "nicolas"